crates/fuel-core/src/schema/da_compressed.rs

```diff
@@ -0,0 +1,51 @@
+use super::{
+    scalars::HexString,
+    ReadViewProvider,
+};
+use crate::{
+    fuel_core_graphql_api::{
+        IntoApiResult,
+        QUERY_COSTS,
+    },
+    query::da_compressed::DaCompressedBlockData,
+    schema::scalars::U32,
+};
+use async_graphql::{
+    Context,
+    Object,
+};
+
+pub struct DaCompressedBlock {
+    bytes: Vec<u8>,
+}
+
+impl From<Vec<u8>> for DaCompressedBlock {
+    fn from(bytes: Vec<u8>) -> Self {
+        Self { bytes }
+    }
+}
+
+#[Object]
+impl DaCompressedBlock {
+    async fn bytes(&self) -> HexString {
+        HexString(self.bytes.clone())
+    }
+}
+
+#[derive(Default)]
+pub struct DaCompressedBlockQuery;
+
+#[Object]
+impl DaCompressedBlockQuery {
+    #[graphql(complexity = "2 * QUERY_COSTS.storage_read + child_complexity")]
```

Comment:
This I still have to figure out. Reviewers are free to point out solutions.

Reply:
Maybe we want to have a new cost just for the compressed block since it doesn't have any children. Something like `1000` for compressed block

Reply:
e165e9e533